client/packages/android/app/src/main/java/org/openmsupply/client/NativeApi.java

```diff
@@ -341,23 +345,20 @@ public void sendTabKeyPress(PluginCall call) {
                 eventTime,
                 KeyEvent.ACTION_DOWN,
                 KeyEvent.KEYCODE_TAB,
-                0
-        );
+                0);
```

Comment:
I think you'll avoid a lot of these formatting issues if you save files with Android Studio, although I am not fully sure about that